packages/dataviews/src/view-list.tsx

```diff
@@ -318,7 +316,7 @@ export default function ViewList< Item extends AnyItem >(
 	} = props;
 	const baseId = useInstanceId( ViewList, 'view-list' );
 	const selectedItem = data?.findLast( ( item ) =>
-		selection.includes( item.id )
+		selection.includes( getItemId( item ) )
```

Comment:
I guess that's a bug fix.

Reply:
Yes. Another little suspicious thing is the optional chaining in `data?.findLast`. The types say that `data` is mandatory. If that's really true, the optional chaining is not needed. But that's a very minor issue compared with other problems.